fastestimator/network.py

```diff
@@ -661,6 +664,11 @@ def _generate_model_names(num_names):
 
     if not hasattr(build, "count"):
         build.count = 0
+    # mix-precision handling
+    if mixed_precision:
+        mixed_precision_tf.set_policy(mixed_precision_tf.Policy('mixed_float16'))
+    else:
+        mixed_precision_tf.set_policy(mixed_precision_tf.Policy('float32'))
```

Comment:
if the second call to fe.build() sets the policy back to 'float32' will it still used mixed precision on the earlier models?

Reply:
yep